### ISSUE TITLE
cli parameters are not being used

### ISSUE THREAD

**Issue by rjanjua** [elderfo/react-native-storybook-loader#31]
I am using version 1.3.2 of this package. When I type in the following command:

`node ./node_modules/.bin/rnstl --searchDir ./src ./packages --pattern **/*.custom.pattern.js --outputFile ./custom-file.js`

I get the following logs for output and patterns:

```
Generating Dynamic Storybook File List

Output file:       /path/to/my/module/storybook/storyLoader.js
Patterns:          ["/path/to/my/module/storybook/stories/index.js"]

```

The output file and pattern being used are the default.

I tried it with node versions 6.9.1 and 8.1.2

**Comment by elderfo** [comment]
I have confirmed this issue and am looking into it. Thanks for the report!

**Comment by elderfo** [comment]
Fixed in [1.4.0](https://github.com/elderfo/react-native-storybook-loader/releases/tag/v1.4.0)

Note: this fix introduces breaking changes. 

CLI can now be accessed from a terminal 
```bash
./node_modules/.bin/rnstl <options>
```
or in package.json 
```json
{
  "scripts": {
    "prestorybook": "rnstl <options>"
  }
}
```

_Note:_ When using a glob with `**/*` it is required to be wrapped in quotes

There is no longer a need to use `node ./node_modules/.bin/rnstl
<options>`.
